CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Changelog
 
 # Unreleased
+* remove matter-labs BLS12-381 implementation, in favor of gnark and constantine implementations [#225](https://github.com/hyperledger/besu-native/pull/225)
```

Comment:
Since a feature is being removed, consider a major version number bump.

Reply:
how about a minor rev to 1.1.0 ?  since we just went 1.0 for the pectra release, it seems out-of-cadence to bump to 2.0.0